broker/src/main/java/org/apache/rocketmq/broker/processor/SendMessageProcessor.java

```diff
@@ -190,6 +195,11 @@ private RemotingCommand consumerSendMsgBack(final ChannelHandlerContext ctx, fin
                 response.setRemark("topic[" + newTopic + "] not exist");
                 return response;
             }
+            String msgId = msgExt.getProperty(MessageConst.PROPERTY_UNIQ_CLIENT_MESSAGE_ID_KEYIDX);
+            if (msgId == null) {
+                msgId = msgExt.getProperty(MessageConst.PROPERTY_ORIGIN_MESSAGE_ID);
+            }
+            dlqLogger.info("[DLQ] topic:" + retryTopic + " consumerGroup:" + requestHeader.getGroup() + " msgId:" + msgId);
```

Comment:
Can we use the format version of info? like: `log.info("Topic: {}", topic)`?

Reply:
Thx,This problem has been fixed.